components/ Subcategory Card/SubCatCard.js

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+const SubCatCard = ({ backgroundColour, title, category, link }) => {
+  console.log(backgroundColour);
+  return (
+    <div>
+      <Link href={link}>
+        <div
+          className={
+            "w-64 md:w-96 h-24 md:h-48 m bg-" +
+            backgroundColour +
+            "-300 rounded-2xl flex flex-col justify-end text-white"
+          }
+        >
+          <h3 className="  text-base md:text-2xl block text-white">
+            {category}
+          </h3>
+          <h4 className="text-2xl md:text-4xl font-bold text-white pb-10">{title}</h4>
+          <a></a>
```

Comment:
What's the meaning behind this a tag?

Reply:
This is obviously the wrong place for it, but should I leave the <a> tag(relocate it to where <link> tags are) for accessibilityof the page or remove it?

Reply:
The current change is good 👍